draft-ietf-httpbis-http2.xml

```diff
@@ -2354,6 +2348,13 @@ HTTP2-Settings    = token68
           the identified stream is the highest numbered stream initiated by the client.
         </t>
         <t>
+          Once sent, the sender will ignore frames sent on streams initiated by the receiver if the

```

Comment:
This is simply moved text.  I think that a statement of fact rather than requirement is best.